components/form/otpVerificationForm.tsx

```diff
@@ -0,0 +1,72 @@
+import { Box, Button, TextField } from '@mui/material';
+import Grid from '@mui/material/Grid';
+import * as React from 'react';
+
+function OtpVerificationForm() {
+    return (
+        <Box
+            sx={{
+                display: 'flex',
+                justifyContent: 'center',
```

Comment:
@JRakin can we not use inline css.

Reply:
@sifatul vaiya this will all be removed later when Ibrahim vai will be finished creating the reusable component I just designed a minimal UI